src/Symfony/Component/Debug/Tests/Fixtures/FinalClass.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Debug\Tests\Fixtures;
+
+/**
+ * @final since version 3.3.
+ */
+class FinalClass
```

Comment:
it's not needed to trigger warning to the user that the class will be final if it already is.